app/src/main/java/com/hedvig/app/feature/marketpicker/LanguageRepository.kt

```diff
@@ -1,29 +1,25 @@
 package com.hedvig.app.feature.marketpicker
 
-import com.apollographql.apollo3.ApolloCall
 import com.apollographql.apollo3.ApolloClient
-import com.apollographql.apollo3.api.ApolloResponse
-import com.apollographql.apollo3.exception.ApolloException
 import com.hedvig.android.owldroid.graphql.UpdateLanguageMutation
-import com.hedvig.android.owldroid.type.Locale
+import com.hedvig.android.owldroid.graphql.type.Locale
 import e
 import i
 
 class LanguageRepository(
     private val apolloClient: ApolloClient,
 ) {
 
-    fun uploadLanguage(acceptLanguage: String, locale: Locale) {
+    suspend fun uploadLanguage(acceptLanguage: String, locale: Locale) {
```

Comment:
This mutation currently discards any errors (because it's really not that important), but the new implementation doesn't seem to mirror this fact (unless `execute` has guarantees that it doesn't throw exceptions?)